### PR TITLE
Blockout file structure

### PR DESCRIPTION
**Pull request by rocktavious**

### CODE REVIEW

File: examples/simple/versions.tf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.0"
```
Comment (davidbloss) [inline examples/simple/versions.tf:2]:
```suggestion
  required_version = ">= 1.7"
```
Let's bump this to match [the min reqs we advertise](https://github.com/opslevel/terraform-provider-opslevel/?tab=readme-ov-file#minimum-requirements)

File: modules/team/versions.tf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.0"
```
Comment (davidbloss) [inline modules/team/versions.tf:2]:
```suggestion
  required_version = ">= 1.7"
```
Let's bump this to match [the min reqs we advertise](https://github.com/opslevel/terraform-provider-opslevel/?tab=readme-ov-file#minimum-requirements)

File: workspace/.terraform.lock.hcl
Comment (davidbloss) [inline workspace/.terraform.lock.hcl:1]: This should be generated each `task plan` or `task apply` right? Should this be added to `.gitignore`?

File: workspace/versions.tf [hunk 3]
```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.0"
```
Comment (davidbloss) [inline workspace/versions.tf:2]:
```suggestion
  required_version = ">= 1.7"
```
Let's bump this to match [the min reqs we advertise](https://github.com/opslevel/terraform-provider-opslevel/?tab=readme-ov-file#minimum-requirements)

File: modules/team/main.tf [hunk 4]
```diff
@@ -0,0 +1,17 @@
+data "opslevel_teams" "all" {}
+
+resource "opslevel_team" "test" {
+  aliases          = var.aliases
+  name             = var.name
+  parent           = var.parent
+  responsibilities = var.responsibilities
+
+  # Iterate over the list of members to create member blocks
+  dynamic "member" {
```
Comment (davidbloss) [inline modules/team/main.tf:10]: neat! ✨

**Review by davidbloss — APPROVED** [review]
Left a few comments but nothing blocking :shipit:

File: workspace/.terraform.lock.hcl
Comment (rocktavious) [reply workspace/.terraform.lock.hcl:1]: According to Github - no.  Its like a yarn.lock file that you would commit.

**Pull request merged**